### PR TITLE
add way to use SQL for checkpoints instead of couch

### PR DESCRIPTION
**Pull request by czue**
gonna throw this on staging for a bit.

@snopoke cc @calellowitz

### CODE REVIEW

File: pillowtop/checkpoints/manager.py [hunk 1]
```diff
@@ -40,20 +46,21 @@ def __init__(self, dao, checkpoint_id):
...
         self._last_checkpoint = checkpoint
-        return checkpoint
+        return result
 
     def update_to(self, seq):
         pillow_logging.info(
             "(%s) setting checkpoint: %s" % (self.checkpoint_id, seq)
         )
-        checkpoint = self.get_or_create(verify_unchanged=True)
+        checkpoint = self.get_or_create(verify_unchanged=True)[0]
```
Comment (snopoke) [inline pillowtop/checkpoints/manager.py]: `.document` would be clearer

Comment (czue) [inline pillowtop/checkpoints/manager.py]: thanks, updated

**Comment by snopoke** [comment]
:+1: once tests pass

**Comment by czue** [comment]
this should be good as soon as tests pass here and on https://github.com/dimagi/commcare-hq/pull/8957

**Pull request merged**